pages/common/dd.md

```diff
@@ -3,26 +3,22 @@
 > Convert and copy a file.
 > More information: <https://www.gnu.org/software/coreutils/dd>.
 
-- Make a bootable USB drive from an isohybrid file (such like `archlinux-xxx.iso`):
+- Make a bootable USB drive from an isohybrid file (such like `archlinux-xxx.iso`) and show progress:
 
-`dd if={{path/to/file.iso}} of=/dev/{{usb_drive}}`
+`dd status=progress if={{path/to/file.iso}} of=/dev/{{usb_drive}}`
 
-- Clone a drive to another drive with 4 MiB block and ignore error:
+- Clone a drive to another drive with 4 MiB block size and flush writes before the command terminates:
 
-`dd if=/dev/{{source_drive}} of=/dev/{{dest_drive}} bs={{4194304}} conv={{noerror}}`
+`dd if=/dev/{{source_drive}} of=/dev/{{dest_drive}} bs={{4M}} conv={{fsync}}`
```

Comment:
- We can't use "M" in the common page, from the [GNU manual page](https://www.gnu.org/software/coreutils/manual/html_node/dd-invocation.html):
> The numeric-valued strings above (n and bytes) are unsigned decimal integers that can be followed by a multiplier: ‘b’=512, ‘c’=1, ‘w’=2, ‘xm’=m, or any of the standard block size suffixes like ‘k’=1024 (see Block size). These multipliers are GNU extensions to POSIX, except that POSIX allows bytes to be followed by ‘k’, ‘b’, and ‘xm’. Note ‘xm’ can be used more than once in a number. Block sizes (i.e., specified by bytes strings) must be nonzero.

- "fsync" is not a possible value for conv for the POSIX specification:
> conv=value[,value ...]
>                 Where values are <comma>-separated symbols from the following
>                  list:
> 
>                  ascii    Convert EBCDIC to ASCII; see Table 4-7, ASCII to
>                           EBCDIC Conversion.
> 
>                  ebcdic   Convert ASCII to EBCDIC; see Table 4-7, ASCII to
>                           EBCDIC Conversion.
> 
>                  ibm      Convert ASCII to a different EBCDIC set; see Table
>                           4-8, ASCII to IBM EBCDIC Conversion.
> 
>                  The ascii, ebcdic, and ibm values are mutually-exclusive.
> 
>                  block    Treat the input as a sequence of <newline>-terminated
>                           or end-of-file-terminated variable-length records
>                           independent of the input block boundaries. Each record
>                           shall be converted to a record with a fixed length
>                           specified by the conversion block size. Any <newline>
>                           shall be removed from the input line; <space>
>                           characters shall be appended to lines that are shorter
>                           than their conversion block size to fill the block.
>                           Lines that are longer than the conversion block size
>                           shall be truncated to the largest number of characters
>                           that fit into that size; the number of truncated lines
>                           shall be reported (see the STDERR section).
> 
>                           The block and unblock values are mutually-exclusive.
> 
>                  unblock  Convert fixed-length records to variable length. Read
>                           a number of bytes equal to the conversion block size
>                           (or the number of bytes remaining in the input, if
>                           less than the conversion block size), delete all
>                           trailing <space> characters, and append a <newline>.
> 
>                  lcase    Map uppercase characters specified by the LC_CTYPE
>                           keyword tolower to the corresponding lowercase
>                           character. Characters for which no mapping is
>                           specified shall not be modified by this conversion.
> 
>                           The lcase and ucase symbols are mutually-exclusive.
> 
>                  ucase    Map lowercase characters specified by the LC_CTYPE
>                           keyword toupper to the corresponding uppercase
>                           character. Characters for which no mapping is
>                           specified shall not be modified by this conversion.
> 
>                  swab     Swap every pair of input bytes.
> 
>                  noerror  Do not stop processing on an input error. When an
>                           input error occurs, a diagnostic message shall be
>                           written on standard error, followed by the current
>                           input and output block counts in the same format as
>                           used at completion (see the STDERR section). If the
>                           sync conversion is specified, the missing input shall
>                           be replaced with null bytes and processed normally;
>                           otherwise, the input block shall be omitted from the
>                           output.
> 
>                  notrunc  Do not truncate the output file. Preserve blocks in
>                           the output file not explicitly written by this
>                           invocation of the dd utility. (See also the preceding
>                           of=file operand.)
> 
>                  sync     Pad every input block to the size of the ibs= buffer,
>                           appending null bytes. (If either block or unblock is
>                           also specified, append <space> characters, rather than
>                           null bytes.)

Reply:
got it. change reverted in [3c96ff4](https://github.com/tldr-pages/tldr/pull/12442/commits/3c96ff4308ca72883ea7c297515c7079e3dd82f7)